src/components/BalanceGame/AnimatedText.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from "react";
+import { AnimatePresence, motion } from "framer-motion";
+
+interface AnimatedTextProps {
+  children: React.ReactNode;
+  // 애니메이션 트리거 값 (예: 클릭한 옵션 여부)
+  animateTrigger: boolean;
+}
+
+function AnimatedText(props: AnimatedTextProps) {
```

Comment:
```suggestion
function AnimatedText({ animateTrigger, children }: AnimatedTextProps) {
```
원래는 구조 분해 할당 사용하셨던 것 같은데 여기는 props 객체로 사용하신 이유가 있으신가요??

Reply:
아뇨 자동완성 때문에; :) @wo-o29